### PR TITLE
New version: github.com.benny93.kafui 0.1.23

### PR DESCRIPTION
**Pull request by Benny93**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/147569)

### CODE REVIEW

**Comment by Benny93** [comment]
@microsoft-github-policy-service agree company="Benny93"

File: manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+PackageIdentifier: github.com.benny93.kafui
```
Comment (stephengillie) [inline manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.installer.yaml:3]:
```suggestion
PackageIdentifier: benny93.kafui
```
Please update file path (between `/b/` and `/kafui/`) to match this change.

File: manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+PackageIdentifier: github.com.benny93.kafui
```
Comment (stephengillie) [inline manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: benny93.kafui
```
Please update file path (between `/b/` and `/kafui/`) to match this change.

File: manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+PackageIdentifier: github.com.benny93.kafui
```
Comment (stephengillie) [inline manifests/b/Benjamin Vollmer/kafui/0.1.23/github.com.benny93.kafui.yaml:3]:
```suggestion
PackageIdentifier: benny93.kafui
```
Please update file path (between `/b/` and `/kafui/`) to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Benny93,

Is the reduced `PackageIdentifier` acceptable? Having just your username might make the package easier to find. 
- The file paths have to include the `PackageIdentifier`, so it should replace your name. This can be through creating a new folder and moving, or renaming the existing folder.

**Comment by stephengillie** [comment]
Automatic Validation ended with: 
> Git checkout failed with exit code: 128

I think it can't run until the paths are fixed.

**Comment by stephengillie** [comment]
> Automatic Validation ended with:
> 
> > Git checkout failed with exit code: 128
> 
> I think it can't run until the paths are fixed.

The situation is unchanged.

**Comment by stephengillie** [comment]
The PR now appears to be adding 4 manifests:

image

We have a limit of 1 manifest change (add/remove/modify) per PR - if this is accurate, could the extra manifests be removed from this PR, and possibly moved to another PR?

**Comment by stephengillie** [comment]
> > Automatic Validation ended with:
> > > Git checkout failed with exit code: 128
> > 
> > 
> > I think it can't run until the paths are fixed.
> 
> The situation is unchanged.

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Packages\benny93.kafui__DefaultSource\kafui.exe returned exit code: 2
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\kafui.exe returned exit code: 2
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Packages\benny93.kafui__DefaultSource\kafui.exe returned exit code: 2
> Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\kafui.exe returned exit code: 2

(Automated response - build 877.)

**Comment by stephengillie** [comment]
Manual Validation ended with: 
```
Current Context: default localhost:9092panic: Unable to get cluster admin: kafka: client has run out of available brokers to talk to: dial tcp [::1]:9092: connectex: No connection could be made because the target machine actively refused it.                                                            <:>: Search resource         <c>: Copy current line
                                                             </>: Search in table         <Enter>: Show details

┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│goroutine 20 [running]:                                                                                               │
└───────────────────────github.com/Benny93/kafui/pkg/datasource/kafds.errorExit(...)───────────────────────────────────┘
╔══════════════════════════════════════════════════════<Topic (0)>══════════════════════/home/runner/work/kafui/kafui/pkg/datasource/kafds/datasource_kaf.go:413                                                                               ║
║                                       github.com/Benny93/kafui/pkg/datasource/kafds.getClusterAdmin()                ║
║                                                                                                       /home/runner/work/kafui/kafui/pkg/datasource/kafds/datasource_kaf.go:372 +0x9f                                                         ║
║                                                             github.com/Benny93/kafui/pkg/datasource/kafds.KafkaDataSourceKaf.GetTopics({})                                                                                                   ║
║                       /home/runner/work/kafui/kafui/pkg/datasource/kafds/datasource_kaf.go:38 +0x1f                  ║
║                                                                                                    github.com/Benny93/kafui/pkg/kafui.(*ResouceTopic).FetchTopics(0xc000289350, {0x16151a0?, 0x1c57d00?})                                    ║
║                                                                                       /home/runner/work/kafui/kafui/pkg/kafui/resource_topic.go:52 +0x26                                                                                     ║
║                                 github.com/Benny93/kafui/pkg/kafui.(*ResouceTopic).UpdateTableDataRoutine.func1()    ║
║                                                                                                                      /home/runner/work/kafui/kafui/pkg/kafui/resource_topic.go:36 +0x6a                                                      ║
║                                                                created by github.com/Benny93/kafui/pkg/kafui.(*ResouceTopic).UpdateTableDataRoutine in goroutine 1                                                                           ║
║                                               /home/runner/work/kafui/kafui/pkg/kafui/resource_topic.go:32 +0x74     ║
║
```

Hi @Benny93,

Is this expected? Is a dependency missing?

**Comment by Benny93** [comment]
> Manual Validation ended with:
> 
> ```
> Current Context: default localhost:9092panic: Unable to get cluster admin: kafka: client has run out of available brokers to talk to: dial tcp [::1]:9092: connectex: No connection could be made because the target machine actively refused it.                                                            <:>: Search resource         <c>: Copy current line
>                                                              </>: Search in table         <Enter>: Show details
> 
> ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
> │goroutine 20 [running]:                                                                                               │
> └───────────────────────github.com/Benny93/kafui/pkg/datasource/kafds.errorExit(...)───────────────────────────────────┘
> ╔══════════════════════════════════════════════════════<Topic (0)>══════════════════════/home/runner/work/kafui/kafui/pkg/datasource/kafds/datasource_kaf.go:413                                                                               ║
> ║                                       github.com/Benny93/kafui/pkg/datasource/kafds.getClusterAdmin()                ║
> ║                                                                                                       /home/runner/work/kafui/kafui/pkg/datasource/kafds/datasource_kaf.go:372 +0x9f                                                         ║
> ║                                                             github.com/Benny93/kafui/pkg/datasource/kafds.KafkaDataSourceKaf.GetTopics({})                                                                                                   ║
> ║                       /home/runner/work/kafui/kafui/pkg/datasource/kafds/datasource_kaf.go:38 +0x1f                  ║
> ║                                                                                                    github.com/Benny93/kafui/pkg/kafui.(*ResouceTopic).FetchTopics(0xc000289350, {0x16151a0?, 0x1c57d00?})                                    ║
> ║                                                                                       /home/runner/work/kafui/kafui/pkg/kafui/resource_topic.go:52 +0x26                                                                                     ║
> ║                                 github.com/Benny93/kafui/pkg/kafui.(*ResouceTopic).UpdateTableDataRoutine.func1()    ║
> ║                                                                                                                      /home/runner/work/kafui/kafui/pkg/kafui/resource_topic.go:36 +0x6a                                                      ║
> ║                                                                created by github.com/Benny93/kafui/pkg/kafui.(*ResouceTopic).UpdateTableDataRoutine in goroutine 1                                                                           ║
> ║                                               /home/runner/work/kafui/kafui/pkg/kafui/resource_topic.go:32 +0x74     ║
> ║
> ```
> 
> Hi @Benny93,
> 
> Is this expected? Is a dependency missing?

Thank you for your review! It appears that the script was executed successfully. Typically, the script connects to a Kafka broker. However, in this instance, since there is none available, it simply panics. This is an early release, and subsequent versions will handle this situation more gracefully. In my testing, when a Kafka broker is configured, it functions as expected.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**